README.md

```diff
@@ -26,14 +26,34 @@ This guide provides instructions on how to spin up a new localnet using the Elys
 
    This command will install the `elysd` daemon.
 
-3. **Download the Latest Snapshot**
+3. **Download the Latest TestNet Snapshot**
 
-   Get the latest snapshot available for the Elys network by using the following command:
+   Get the latest TestNet snapshot available for the Elys network.
+
+   **Main branch**
+
+   Use the following command to download the latest TestNet snapshot that uses the changes from the `main` branch:
 
    ```bash
    rm -rf ~/.elys && curl -o - -L https://snapshots.elys.network/elys-snapshot-main.tar.lz4 | lz4 -c -d - | tar -x -C ~/
    ```
 
+   **Devnet branch**
+
+   Use the following command to download the latest TestNet snapshot that uses the changes from the `devnet` branch:
+
+   ```bash
+   rm -rf ~/.elys && curl -o - -L https://snapshots.elys.network/elys-snapshot-devnet.tar.lz4 | lz4 -c -d - | tar -x -C ~/
```

Comment:
<img width="755" alt="Screenshot 2024-06-19 at 7 49 01 PM" src="https://github.com/elys-network/bindings/assets/26249903/dc8334c2-405a-4fe8-8dbe-33603264a76e">

Reply:
`develop` is working fine

Reply:
@amityadav0 we have to wait for changes within the `devnet` branch in the elys repo to trigger the construction of the snapshot for `devnet`